Assets/Scripts/TileMapDemoScripts/BaseTile.cs

```diff
@@ -16,14 +16,32 @@ public class BaseTile : Tile
     protected bool isCollision;
     public BaseTile parent;
 
+    //A* Pathfinding Info
+    public List<BaseTile> neighbors = new List<BaseTile>();
+    public int fCost, gCost, hCost = 0;
+
     public override void GetTileData(Vector3Int position, ITilemap tilemap, ref TileData tileData)
     {
         base.GetTileData(position, tilemap, ref tileData);
+
+        x = 0; y = 0;
```

Comment:
This entire section is copy and pasted from some below code. 

A) A bracket is left out which is causing a compilation error.
B) Like in C++, in C# you can't declare functions with the same parameters multiple times so this still wouldn't compile.